apps/prairielearn/src/pages/administratorFeatures/administratorFeatures.sql

```diff
@@ -27,20 +27,26 @@ ORDER BY
 SELECT
   *
 FROM
-  institutions;
+  institutions
+ORDER BY
+  long_name ASC;
```

Comment:
I try and always finish an ORDER BY list with something that is guaranteed non-NULL and unique, to ensure a stable order. `id` would be the obvious choice here.

Reply:
And for what it's worth, I don't explicitly write ASC because it's the default. I find that only writing DESC when necessary makes it clearer when we are using a non-default order.

Reply:
I can add `id` as a secondary column 👍 

I'm largely indifferent about including `ASC`. I guess I tend to do it to be explicit, which may help folks who don't have it ingrained that `ASC` is default (which is myself half the time haha). I can remove it if you feel strongly.

Reply:
Changed in bd1f387.